<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget
{
	background-color: rgb(85, 85, 127);
}</string>
    </property>
    <widget class="QPushButton" name="m_pCircleBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton 
{
    border: none;
  
	image: url(:/image/ellipse_default.png);
}

QPushButton:hover 
{
  border: none;
  
  image: url(:/image/ellipse_hover.png);
}

QPushButton:pressed 
{
    border: none;
	
	image: url(:/image/ellipse_selected.png);
}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="m_pRectBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton 
{
    border: none;
  
	
	image: url(:/image/rect_default.png);
}

QPushButton:hover 
{
  border: none;
  
 
	image: url(:/image/rect_hover.png);
}

QPushButton:pressed 
{
    border: none;
	
	
	image: url(:/image/rect_selected.png);
}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="m_pLineBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton 
{
    border: none;
    
	image: url(:/image/line_default.png);
	
	
}

QPushButton:hover 
{
  border: none;
  
 
	
	image: url(:/image/line_hover.png);
}

QPushButton:pressed 
{
    border: none;
	
	
	
	image: url(:/image/line_selected.png);
}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="m_pHexBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton 
{
    border: none;
    
	image: url(:/image/peiqi.png);
	

}

QPushButton:hover 
{
  border: none;
  
 
	
	image: url(:/image/peiqi.png);
}

QPushButton:pressed 
{
    border: none;
	
	
	
	image: url(:/image/peiqi.png);
}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="KxCanvas" name="m_pcanvas" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>681</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 0);
</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KxCanvas</class>
   <extends>QWidget</extends>
   <header>kxcanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
